CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 # Version 3.37.0 (2023-xx-xx)
+## Changed
```

Comment:
@karenkyang @whistler are we okay with this being a breaking change?

Reply:
Ignore this. I saw that we're setting name = instruction in Classification class!